app/javascript/packs/Restaurant.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types'
+
+class Restaurant extends React.Component {
+
+    render() {
+        const {restaurant} = this.props;
+        let tenbis = "";
+        if (restaurant.tenbis) {
```

Comment:
in render: `restaurant.tenbis && <img src="../../assets/tenbis_img.jpg" height="40" width="40"/>`